pegasus/sites.v3/code.org/views/educate_box.haml

```diff
@@ -1,4 +1,5 @@
 -color ||= "#59b9dc"
+-item[:button_s] ||= 'Learn more'
```

Comment:
Oh, button_s was one of the columns of the gsheet so each box could have a different button label. But they were all `Learn more` so instead of transferring that data into the curriculum_boxes and resources_boxes rows I made it the default.